src/material/checkbox/checkbox.ts

```diff
@@ -380,6 +380,9 @@ export abstract class _MatCheckboxBase<E>
         });
       }
 
+      // Normally the input should be focused already, but if the click comes from something
+      // like the touch target, then we might have to focus it ourselves.
+      // this._inputElement.nativeElement.focus();
```

Comment:
It seems you have just added comments.

Reply:
ops, thanks for flagging it. I was in a rush and forgot to re-enable the fix.